src/Komiku/Komiku.ts

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable linebreak-style */
+import {
+  LanguageCode,
+  SourceInfo,
+  TagType,
+  ContentRating,
+} from "paperback-extensions-common";
+
+import { MangaStream, getExportVersion } from "../MangaStream";
+
+import { KomikuParser } from "./KomikuParser";
+
+const KOMIKU_DOMAIN = "https://komiku.com";
+
+export const KomikuInfo: SourceInfo = {
+  version: getExportVersion("0.0.6"),
```

Comment:
Why start with 0.0.6?

Reply:
Oh, i thought it was 0.0.6 version lol my bad